src/main.rs

```diff
@@ -163,6 +126,7 @@ fn main() {
         .arg("-a".to_owned())
         .arg("--delete")
         .arg("--compress")
+        .arg(format!("-e ssh -p {}", conf.remote.ssh_port))
```

Comment:
Are you sure this will work? This seems to me as if it would be treated as one string argument.

Reply:
I changed it to `args`. But it is still untested as I currently don't have adequate access to my infrastructure.

Reply:
Fixed it to [this](https://explainshell.com/explain?cmd=rsync+-e).